src/index.ts

```diff
@@ -87,8 +91,8 @@ function defaultFilterEnv(key: string): boolean {
   return APP_SPECIFIC_ENV.test(key)
 }
 
-const truthy = new Set([ "y", "yes", "true", true ])
-const falsy = new Set([ "n", "no", "false", false ])
+const truthy = new Set(["y", "yes", "true", true])
+const falsy = new Set(["n", "no", "false", false])
```

Comment:
Seems like this was the result of executing prettier, but not eslint. There is a "format" script to use :)